aspnetcore/fundamentals/configuration/index.md

```diff
@@ -59,6 +59,8 @@ var builder = WebApplication.CreateBuilder(args);
 1. [appsettings.json](#appsettingsjson) using the [JSON configuration provider](#jcp).
 1. A fallback to the host configuration described in the [next section](#host).
 
+Note: `WebApplication.CreateBuilder(args)` should only be called once.
```

Comment:
```suggestion
```

Many of our docs use the same API many times, for example `app.UseCors` `https://learn.microsoft.com/en-us/aspnet/core/security/cors

Reply:
The link we originally had here tells why you don't want to call this API multiple times but people tend to think it can or needs to be. Perhaps it would be good to put the link back in?
https://techcommunity.microsoft.com/blog/iis-support-blog/asp-net-core-503-server-has-been-shutdown/3830338

Reply:
Sure